sklearn/linear_model/_sag_fast.pyx.tp

```diff
@@ -247,11 +247,9 @@ def sag{{name_suffix}}(SequentialDataset{{name_suffix}} dataset,
     https://hal.inria.fr/hal-00860051/document
     (section 4.3)
 
-    Defazio, A., Bach, F., Lacoste-Julien, S. (2014),
-    SAGA: A Fast Incremental Gradient Method With Support
-    for Non-Strongly Convex Composite Objectives
-    https://arxiv.org/abs/1407.0202
-
+    :arxiv: `Defazio, A., Bach F. & Lacoste-Julien S. (2014).
+    "SAGA: A Fast Incremental Gradient Method With Support
+    for Non-Strongly Convex Composite Objectives" <1407.0202>`
```

Comment:
This will solve your problem.
```suggestion
    :arxiv:`Defazio, A., Bach F. & Lacoste-Julien S. (2014).
    "SAGA: A Fast Incremental Gradient Method With Support
    for Non-Strongly Convex Composite Objectives" <1407.0202>`
```